### PR TITLE
[Editors Page] Add LSP to VS Code section

### LINKED ISSUE #545: [Editors Page] Add LSP to VS Code section
Since the VS Code extension, will support Language Server Protocol in the next time, 
we should add some info about this to the corresponding section in the [Editors Page](https://github.com/openhab/openhab-docs/blob/gh-pages/configuration/editors.md) 

Especially the requirements should be mentioned.
- Correct Server configuration in VS Code
- LSP AddOn installed with Paper UI

Related:
[VS Code Repo Readme](https://github.com/openhab/openhab-vscode#validating-the-rules)
[Corresponding Question in the community](https://community.openhab.org/t/visual-studio-code-openhab-extension/30205/104?u=confectrician)

We should finish the PR and merge it when those changes are in the stable build.

@kubawolanin do you want to add it yourself?
I can do that PR too.
If you have any wishes about the wording, throw it in here.

I would mainly use your Readme text and add one or two sentences for the users to explain the benefits from LSP for them and what this is.

### PR DESCRIPTION
**Pull request by Confectrician**
Closes #545 
Changes as discussed in the issue.

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

**Comment by kubawolanin** [comment]
LGTM

File: configuration/editors.md [hunk 1]
```diff
@@ -19,25 +19,24 @@ This documentation page can give you some guidance in choosing the right one for
...
-    - Open command line in the same folder you downloaded the extension
-      Type the following command:
-      ```shell
-        code --install-extension openhab-0.0.2.vsix
-      ```
-    - Go to the extensions area and install the extension like showed in the screenshot:
-      ![openHAB VS Code Extension alternative installation](images/vscode_alternative_install.png)
-
-Currently openhab-vscode isn't yet listed in [Microsoft Visual Studio Marketplace](https://marketplace.visualstudio.com/vscode), but the contributors are working on it already.
+1. Open the extension sidebar. ![openHAB VS Code Extension alternative installation](images/vscode_extensiontab_icon.png)
```
Comment (ThomDietrich) [inline configuration/editors.md]: Should the step 0 be "Install VS Code if not yet available"?

Comment (kubawolanin) [reply configuration/editors.md]: `0. Install [Visual Studio Code](https://code.visualstudio.com/Download) if not yet available` ;-)

**Comment by Confectrician** [comment]
> Should the step 0 be "Install VS Code if not yet available"?

No problem. Done.

> LGTM

I think we can merge then. :)

**Comment by ThomDietrich** [comment]
Wonderful! Thanks ;)

**Pull request merged**